testsuite/src/it/java/io/quarkus/ts/startstop/utils/URLContent.java

```diff
@@ -9,17 +12,21 @@ public enum URLContent {
             new String[]{"http://localhost:8080", "Hello from a simple Jakarta REST app."}
     }),
     FULL_MICROPROFILE(new String[][]{
+            new String[]{"http://localhost:8080/q/health/ready", "\"UP\""},
             new String[]{"http://localhost:8080/data/hello", "Hello World"},
             new String[]{"http://localhost:8080", "Hello from a full MicroProfile suite"},
```

Comment:
`q/health/ready` is a good strategy, we can move this one line above so `data/hello` is close to the other data lines

Reply:
sure, it will revert https://github.com/quarkus-qe/quarkus-startstop/pull/376 but that is fine considering the health probe. will do